pkg/controller/plan/migration.go

```diff
@@ -1405,6 +1449,62 @@ func (r *Migration) updateConversionProgress(vm *plan.VMStatus, step *plan.Step)
 	return
 }
 
+func (r *Migration) updateConversionProgressEl9(pod *core.Pod, step *plan.Step) (err error) {
+	if pod.Status.PodIP == "" {
+		return
+	}
+
+	var disk_re = regexp.MustCompile(`v2v_disk_transfers\{disk_id="(\d+)"\} (\d{1,3}\.?\d*)`)
+	url := fmt.Sprintf("http://%s:2112/metrics", pod.Status.PodIP)
+	resp, err := http.Get(url)
```

Comment:
I think that's ok for the time being but we need to think what's the API we'd like to have between forklift-controller and the populator controller/pod - I think it would eventually make sense to let forklift-controller post the CR (e.g., OvirtVolumePopulator) and then only look at it, so the populator controller would fetch the progress from the populator pod and update the CR (or PVC).

we don't have a CR for virt-v2v so we anyway can't do it this way now, but if that's what we have in mind (and I'm not saying we do, we barely talked about it yet) then we should comment that this is a "temporary" solution that is meant to change

Reply:
this should also be handled separately, resolving for now